node_tree.py

```diff
@@ -307,6 +478,46 @@ def draw_color(self, context, node):
         return self.nodule_color
 
 
+class SvLinkMatrix(bpy.types.Operator):
```

Comment:
well.. i think that's a separate issue (spawning non-overlap) 

don't get stuck on small usability details in a large PullRequest.